addons/account/models/chart_template.py

```diff
@@ -279,6 +279,13 @@ def _load(self, sale_tax_rate, purchase_tax_rate, company):
         # If the floats for sale/purchase rates have been filled, create templates from them
         self._create_tax_templates_from_rates(company.id, sale_tax_rate, purchase_tax_rate)
 
+        # Set the fiscal country before generating taxes in case the company does not have a country_id set yet
+        if company.country_id and not self.country_id:
+            # Otherwise, if there is a country set on the company, set the default fiscal country of the company equal to the company country.
+            company.account_fiscal_country_id = company.country_id
```

Comment:
account_fiscal_country_id is computed, right ? So normally, if company_country_id is set, account_fiscal_country_id should already be == country_id. Am I missing something ?